package.json

```diff
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "description": "Shine blog framework",
   "scripts": {
+    "add-baseurl":"node hugo/lib/getUrl.js",
     "build": "npm run webpack-build && npm run hugo-build",
-    "build-for-staging": "npm run clear-public && npm run clear-content && npm run fetch-content -- --staging && npm run build",
+    "build-for-staging": "npm run clear-public && npm run clear-content && npm run add-baseurl && npm run fetch-content -- --staging && npm run build",
     "build-for-production": "npm run clear-public && npm run clear-content && npm run fetch-content -- --production && npm run build",
```

Comment:
Do you think we should add `npm run add-baseurl` to the `build-for-production` command too?